test/config.js

```diff
@@ -1,13 +1,12 @@
-var Mixpanel = require('../lib/mixpanel-node');
-
-exports.config = {
-    setUp: function(cb) {
-        this.mixpanel = Mixpanel.init('asjdf');
-        cb();
-    },
-
-    "is set to correct defaults": function(test) {
-        test.deepEqual(this.mixpanel.config, {
+const Mixpanel = require('../lib/mixpanel-node');
+
+describe('config', () => {
+    let mixpanel;
+    beforeAll(() => {
```

Comment:
I would also make this one a `beforeEach` - make sure each test case has a fresh mp instance